src/components/nav-bar/nav-bar.component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import NavBarItem from './nav-bar-item/nav-bar-item.component.jsx';
+import OnlineShopButton from './online-shop-button/online-shop-button.component.jsx';
+import './nav-bar.styles.scss';
+
+const NavBar = () => (
+
+    <div className="nav-bar-container">
+        <div className="items">
+            <NavBarItem>Concept</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>Quality</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>Products</NavBarItem>
+        </div>
+        <div className="logo-container">
+            <div className="logo" />
+        </div>
+        <div className="items">
+            <NavBarItem>Shops</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>News</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>Contact</NavBarItem>
+        </div>
+        <div className="shop-button">
```

Comment:
same as the above comment, its not a shop button itself.
choose a more descriptive name.

Reply:
changed to shop-buton-container